### PR TITLE
[fix][broker]Update rest api response class

### PR DESCRIPTION
**Pull request by tuteng**
(cherry picked from commit a594ca71554a5de0e28564d54da6ffc49202206e)

Update response class for rest api

Fixes #xyz

Master Issue: #xyz

### Motivation

The rest api was changed from snyc to async, and all responses became void, which caused the swagger file to not generate the correct response object, the purpose of this pr is to generate the correct response object for the swagger file, which is useful for generating client code automatically using code

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/AmateurEvents/pulsar/pull/34

### CODE REVIEW

**Review by urfreespace — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java [hunk 1]
```diff
@@ -517,7 +526,7 @@ public void deleteMaxUnackedMessagesOnConsumer(@Suspended final AsyncResponse as
 
     @GET
     @Path("/{tenant}/{namespace}/{topic}/deduplicationSnapshotInterval")
-    @ApiOperation(value = "Get deduplicationSnapshotInterval config on a topic.")
+    @ApiOperation(value = "Get deduplicationSnapshotInterval config on a topic.", response = TopicPolicies.class)
```
Comment (nodece) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:529]:
```suggestion
    @ApiOperation(value = "Get deduplicationSnapshotInterval config on a topic.", response = Integer.class)
```

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java [hunk 2]
```diff
@@ -938,7 +947,7 @@ public void getPartitionedMetadata(
 
     @GET
     @Path("/{tenant}/{namespace}/{topic}/properties")
-    @ApiOperation(value = "Get topic properties.")
+    @ApiOperation(value = "Get topic properties.", response = Boolean.class)
```
Comment (nodece) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:950]: Here should be a `Map`.

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java [hunk 3]
```diff
@@ -1188,7 +1201,7 @@ public void getSubscriptions(
 
     @GET
     @Path("{tenant}/{namespace}/{topic}/stats")
-    @ApiOperation(value = "Get the stats for the topic.")
+    @ApiOperation(value = "Get the stats for the topic.", response = TopicStats.class)
```
Comment (nodece) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java:1204]:
```suggestion
    @ApiOperation(value = "Get the stats for the topic.", response = PersistentTopicStats.class)
```

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java (hunk 1 above)
Comment (tuteng) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java]: Fixed

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java (hunk 2 above)
Comment (tuteng) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java]: Fixed

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java (hunk 3 above)
Comment (tuteng) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java]: Fixed

**Review by nodece — APPROVED** [review]

**Pull request merged**